catalog/ui/catalog-ui-search/src/main/webapp/component/singletons/session-timeout.js

```diff
@@ -22,7 +22,7 @@ var invalidateUrl = '/services/internal/session/invalidate?prevurl=';
 var idleNoticeDuration = 60000;
 // Length of inactivity that will trigger user timeout (15 minutes in ms by default)
 // See STIG V-69243
-var idleTimeoutThreshold = parseInt(properties.ui.timeout) > 0 ? parseInt(properties.ui.timeout) : 900000;
+var idleTimeoutThreshold = parseInt(properties.timeout) > 0 ? parseInt(properties.timeout) : 900000;
```

Comment:
Nice catch. I verified that `timeout` is assigned to `props` and not `props.ui` in https://github.com/codice/ddf/blob/master/catalog/ui/catalog-ui-search/src/main/webapp/properties.js#L69
by looking at responses returned by queries to `/search/catalog/internal/config` and `/services/platform/config/ui`